packages/@react-aria/tree/src/useTreeItem.ts

```diff
@@ -46,6 +46,10 @@ export function useTreeItem<T>(props: AriaTreeItemOptions, state: TreeState<T>,
   let gridListAria = useGridListItem(props, state, ref);
   let isExpanded = gridListAria.rowProps['aria-expanded'] === true;
   let stringFormatter = useLocalizedStringFormatter(intlMessages, '@react-aria/tree');
+  let labelProps = useLabels({
+    'aria-label': isExpanded ? stringFormatter.format('collapse') : stringFormatter.format('expand'),
+    'aria-labelledby': gridListAria.rowProps.id
+  });
```

Comment:
for some reason my ipad VO doesn't announce "expand" anymore, just the row label, digging

Reply:
Possibly unhelpful, but it's working on my iPhone
18.3.1

Reply:
thanks for testing in 18! I feel like we've run into this iOS 17 VO before but I couldn't find any prior documentation in our other collapsable/expandable components but will punt for now